test/occupation.jl

```diff
@@ -4,18 +4,29 @@ using SpecialFunctions
 
 include("testcases.jl")
 
+smearing_methods = (
+        DFTK.Smearing.None(),
+        DFTK.Smearing.FermiDirac(),
+        DFTK.Smearing.Gaussian(),
+        DFTK.Smearing.MarzariVanderbilt(),
+        DFTK.Smearing.MethfesselPaxton(1),
```

Comment:
if you want to be super fancy you can do `(DFTK.Smearing.MethfesselPaxton(n) for n=1:4)...` but this is fine too